T4MVCHostMvcApp/T4MVC Files/T4MVC.tt

```diff
@@ -145,14 +145,21 @@ foreach (string folder in settings.StaticFilesFolders
 	) {
     ProcessStaticFiles(Project, folder);
 }
+
+PushIndent("    ");
+#>
+
+[<#= GeneratedCode #>, DebuggerNonUserCode]
+public static partial class Bundles
+{
+<#
+			
+		foreach (string folder in settings.StaticFilesFolders.Concat(GetStaticFilesViewFolders())) {
```

Comment:
Seems your editor is using tabs instead of spaces, so all the new code has it (which then affects generated code as well). Trying to keep it consistently using spaces. Thanks! :)